pkg/apis/okctl.io/v1alpha1/cluster_v1alpha1.go

```diff
@@ -303,15 +306,50 @@ type ClusterDatabasesPostgres struct {
 
 // Validate the content of a postgres database declaration
 func (c ClusterDatabasesPostgres) Validate() error {
+	const minLengthDatabaseName int = 1
+
+	const maxLengthDatabaseName int = 60
+
+	/*
+	 * Validation of database name is based on:
+	 * 1. AWS RDS rules when creating a postgres database
```

Comment:
Would love a link to AWS documentation where they describe the rules

Reply:
well, there is always https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html - but that doesn't explain what the RDS interface actually does (there is more information/restrictions described in the RDS console)

Reply:
Oh right. I thought they'd have a nice name validation list